src/qtism/data/storage/xml/marshalling/PrintedVariableMarshaller.php

```diff
@@ -93,7 +99,11 @@ protected function unmarshall(DOMElement $element)
                 $component->setFormat($format);
             }
 
-            if (Version::compare($version, '2.1.0', '>=') === true && ($powerForm = $this->getDOMElementAttributeAs($element, 'powerForm', 'boolean')) !== null) {
+            if (Version::compare($version, '2.1.0', '>=') === true && ($powerForm = $this->getDOMElementAttributeAs(
+                    $element,
+                    'powerForm',
+                    'boolean'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?